CardanoSharp.Koios.Client/CardanoSharp.Koios.Client.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.1</TargetFramework>
-        <Nullable>enable</Nullable>
-        <Version>1.0.9</Version>
+        <TargetFrameworks>netstandard2.1;netstandard2.0;</TargetFrameworks>
```

Comment:
Going by [this article](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#which-net-standard-version-to-target) you should be able to target `netstandard2.0` only. Which will cover the majority of frameworks (including the ones that are .NET standard 2.1 compatible), or at least until we need newer APIs/dependencies which require the newer framework APIs.